### PR TITLE
add support for the cli itnegration

### PR DESCRIPTION
**Pull request by guillermogutierrez**
[XXXX-<Title> - Please use the Work Item number and Title as PR Name, not subtasks]

#### 📲 What

Support for the stacks cli v2.0 integration

🤔 Why
With the new version of the stacks cli we need to adapt the code to the new requirements.

🛠 How
Stacks cli expects a file called stackscli.yml file with all build steps. Also existing files require minor updates to ensure the scaffolding works as expected

#### 👀 Evidence

Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR

#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: stackscli.yml [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+      desc: Downloading maven dependencies for the cqrs+events project
+
+    # Build archetype for the serenity tests
+    - action: cmd
+      cmd: mvn
+      args: process-resources
+              -f ./api-tests/pom.xml
+              -P archetype-resources
+              --quiet
+      desc: Duplicating the original pom.xml file to archetype-pom.xml
```
Comment (johnsall) [inline stackscli.yml:116]: clarify this is for the tests

File: stackscli.yml [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+              -DpropertyFile=./api-tests/archetype.properties
+              --quiet
+      desc: Creating the archetype resources using the api-test source code
+
+    - action: cmd
+      cmd: mvn
+      args: install
+              -f ./api-tests/target/generated-sources/archetype/pom.xml
+              --quiet
+      desc: Installing the archetype (groupId= com.amido.stacks.tests, artifactId= stacks-api-cqrs-events-tests-archetype, version= 1.0.0) in the local maven repository
```
Comment (johnsall) [inline stackscli.yml:131]: same as above

File: stacksclipom.xml [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+  </profiles>
+
+</project>
```
Comment (johnsall) [inline stacksclipom.xml:118]: add new line

File: stackscli.yml (hunk 1 above)
Comment (guillermogutierrez) [reply stackscli.yml]: Done

File: stackscli.yml (hunk 2 above)
Comment (guillermogutierrez) [reply stackscli.yml:131]: Test references are already included in the artifactId and the groupId, I would keep it as it is

File: stacksclipom.xml (hunk 3 above)
Comment (guillermogutierrez) [reply stacksclipom.xml]: Done

**Review by radoslaw-orlowski — APPROVED** [review]

**Pull request merged**